### ISSUE TITLE
Implement reporting game exception crashes

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive-Launcher#193]
so that users don't need to copy-paste them

Requires: https://github.com/Revolutionary-Games/ThriveDevCenter/issues/523

**Comment by hhyyrylainen** [comment]
Duplicate of: https://github.com/Revolutionary-Games/Thrive-Launcher/issues/163
